### ISSUE TITLE
Servo doesn't work 

### ISSUE THREAD

**Issue by indexdb** [atanisoft/Esp32FeederController#10]
I assembled Max feeder, N20 motro is work, but the Servo doesn't work, I have test the servo, it's good.
everything seem good, but Servo do nothing. I wonder if you test the servo after last bug fixed.

I (4348) feeder_mgr: Initializing I2C Bus
W (4350) feeder_mgr: PCA9685(40) was not detected.
W (4354) feeder_mgr: PCA9685(41) was not detected.
W (4360) feeder_mgr: PCA9685(42) was not detected.
I (4367) feeder_mgr: PCA9685(43/0x3ffca1a8) configured for use.
W (4372) feeder_mgr: PCA9685(44) was not detected.
W (4377) feeder_mgr: PCA9685(45) was not detected.
W (4383) feeder_mgr: PCA9685(46) was not detected.
W (4388) feeder_mgr: PCA9685(47) was not detected.
I (4396) feeder_mgr: Detected PCA9685 devices:1
W (4399) feeder_mgr: MCP23017(20) was not detected!
W (4404) feeder_mgr: MCP23017(21) was not detected!
W (4411) feeder_mgr: MCP23017(22) was not detected!
I (4419) feeder_mgr: MCP23017(23/0x3ffca1e8) configured for use.
W (4422) feeder_mgr: MCP23017(24) was not detected!
W (4428) feeder_mgr: MCP23017(25) was not detected!
W (4433) feeder_mgr: MCP23017(26) was not detected!
W (4439) feeder_mgr: MCP23017(27) was not detected!
I (4444) feeder_mgr: Attempting to create 16 feeders
I (4450) feeder_mgr: Detected 1 PCA9685 and 1 MCP23017
I (4456) feeder_mgr: Creating feeder BA061284 (0/0/0/PCA:0x3ffca1a8/MCP:0x3ffca1e8)
I (4466) feeder: [BA061284:0] Initializing using PCA9685 0x3ffca1a8:0
I (4471) feeder: [BA061284:0] Subscribing to MCP23017 channel 0
I (4478) feeder: [BA061284:0] Feedback enabled using MCP23017 0x3ffca1e8:0
--------------------------------------------------------------------------------------

I (116263) PCA9685: [43:0] Moving to 90 deg
I (116511) feeder: [BA061284:0] Feeder movement complete, turning off servo
I (117223) feeder: [BA061284:0] Feeder fully advanced, retracting
I (117224) PCA9685: [43:0] Moving to 15 deg
I (117467) feeder: [BA061284:0] Feeder movement remaining: 4mm
I (117468) feeder: [BA061284:0] Feeder is retracted
I (117469) feeder: [BA061284:0] Moving to fully advanced position
I (117476) PCA9685: [43:0] Moving to 90 deg
I (117723) feeder: [BA061284:0] Feeder movement complete, turning off servo
I (117912) feeder: [BA061284:0] Feeder fully advanced, retracting
I (117913) PCA9685: [43:0] Moving to 15 deg
I (118156) feeder: [BA061284:0] Feeder movement remaining: 4mm
I (118157) feeder: [BA061284:0] Feeder is retracted
I (118158) feeder: [BA061284:0] Moving to fully advanced position
I (118165) PCA9685: [43:0] Moving to 90 deg
I (118413) feeder: [BA061284:0] Feeder movement complete, turning off servo
I (118702) feeder: [BA061284:0] Feeder fully advanced, retracting
I (118703) PCA9685: [43:0] Moving to 15 deg
I (118946) feeder: [BA061284:0] Feeder movement remaining: 4mm
I (118947) feeder: [BA061284:0] Feeder is retracted
I (118948) feeder: [BA061284:0] Moving to fully advanced position
I (118954) PCA9685: [43:0] Moving to 90 deg

**Comment by atanisoft** [comment]
Did you provide a 5v power to the PCA9685 board? Did the servo move (or make noise) during startup?

**Comment by indexdb** [comment]
of course, +5v is provided to servo, no move no noise anytime.

**Comment by atanisoft** [comment]
Ok, will see if I can reproduce it locally.

**Comment by indexdb** [comment]
I will use oscilloscope to check the pwm signal tomorrow.

**Comment by indexdb** [comment]
I use logic analyzer to check S pin, no signal is captured, I resolder PCA9685, the problem is still the same, servo has no action and noise.

**Comment by indexdb** [comment]
I found the solution. 
This bug is caused by a wrong design of pcb, 23 pin of PCA9685   /OE is low enable. But your circuit diagram connects to 3.3v via 10k resistor, which causes the output to be disabled. The solution is to remove R3 and R9, and will work fine.

**Comment by atanisoft** [comment]
ahh, good find. I somehow didn't see this in my tests. I'll update the PCB to list those as DNP.

**Comment by indexdb** [comment]
I found another issue, the PCA9685 address a0 a1 is connected to high, but a2 a3 a4 a5 is not connected, leading to a random address sometimes instead of 0x43 during i2c scan.
a2 a3 a4 a5 should be properly grounded. Also, the MCP23017 has the same grounding issue, but no problem was found in the actual test. However, I found the MCP23017 connected to the same address on the schematic, i.e. a0 a1 connected to high, a2 not connected.
